### PR TITLE
Fix compiler bootstrapping

### PR DESCRIPTION
**Pull request by cboulanger**

### CODE REVIEW

**Review by johnspackman — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -29,6 +29,28 @@ For more information please see http://qooxdoo.org/LICENCSE .
...
+[NPM release](https://www.npmjs.com/package/@qooxdoo/qx) and a
+compiler called with `npx qx ...`. 
+
+In contrast, when hacking qooxdoo itself, you
+need to first execute the following steps:
+
+```bash
+git clone https://github.com/qooxdoo/qooxdoo.git
+cd qooxdoo
+npm install
```
Comment (hkollmann) [inline README.md:44]: npm ci - then package-lock is not changed

**Review by hkollmann — APPROVED** [review]

**Review by oetiker — APPROVED** [review]
if it was in a branch you could just happily merge way :)

**Comment by cboulanger** [comment]
@oetiker That's true, but I needed the feedback! Much better PR now ... don't worry won't make a PR for each little change.

File: .gitignore [hunk 2]
```diff
@@ -28,10 +28,10 @@ tmp/
 /apps
 /deploy
 /build
-/compiled
```
Comment (derrell) [inline .gitignore:31]: This seems wrong. Why would you want to commit the `comiled` directory?

**Review by derrell — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 2 above)
Comment (cboulanger) [reply .gitignore:31]: This is a duplicate entry. There is already a `compiled` in line 27

**Review by derrell — APPROVED** [review]

**Pull request merged**